software/tracksight/frontend/src/app/components/db/dropdown_menu.tsx

```diff
@@ -1,45 +1,49 @@
-import { useState, useEffect, Dispatch, SetStateAction } from 'react';
-import { Select } from 'antd';
-import React from 'react';
+import { useState, useEffect, Dispatch, SetStateAction } from "react";
+import { Select } from "antd";
+import React from "react";
 
 export interface DropdownMenuProps {
-    setOption: Dispatch<SetStateAction<string[]>>,
-    selectedOptions: string[],
-    options: string[],
-    single: boolean,
-    name: string,
+  setOption: Dispatch<SetStateAction<string[]>>;
+  selectedOptions: string[];
+  options: string[];
+  single: boolean;
+  name: string;
```

Comment:
prettier?

Reply:
sorry I thought this was merged and didn't take a look at the comments, yup! apologies from the prettier changes (wasn't implemented before but I thought it would be a good idea given we're trying to improve code clarity in coming weeks)